.travis.yml

```diff
@@ -39,7 +39,7 @@ install:
   - pipenv install $DJANGO
   # fix issues with travis
   - pipenv install --skip-lock "attrs>=17.4.0"
-  - pipenv install --skip-lock "six>=1.11.0"
+  - pipenv install --skip-lock "six"
```

Comment:
Is travis failing with `"six>=1.11.0"` ?

Reply:
could be, these kind of builds are weird sometimes